lib/rex/ui/text/input/readline.rb

```diff
@@ -36,7 +37,7 @@ def initialize(tab_complete_proc = nil)
     #
     def reset_tab_completion(tab_complete_proc = nil)
       ::Readline.basic_word_break_characters = "\x00"
-      ::Readline.completion_proc = tab_complete_proc || @rl_saved_proc
+      ::Readline.completion_proc = tab_complete_proc ? wrap_tab_compete_proc(tab_complete_proc) : @rl_saved_proc
```

Comment:
Just confirming, when does `reset_tab_completion` get called? Does it make sense to wrap? 🤔

Reply:
It makes sense to wrap because it can overwrite the existing `tab_complete_proc` method, and we want that wrapped in a error handling to avoid it crashing the console